src/containers/collectibles/helpers.ts

```diff
@@ -9,16 +9,19 @@ import { gifPreview } from 'utils/imageProcessingUtil'
  * extensions based on OpenSea metadata standards
  * https://docs.opensea.io/docs/metadata-standards
  */
+const OPENSEA_AUDIO_EXTENSIONS = ['mp3', 'wav', 'oga']
 const OPENSEA_VIDEO_EXTENSIONS = [
   'gltf',
   'glb',
   'webm',
   'mp4',
   'm4v',
   'ogv',
-  'ogg'
+  'ogg',
+  'mov'
 ]
-const OPENSEA_AUDIO_EXTENSIONS = ['mp3', 'wav', 'oga']
+
+const SUPPORTED_VIDEO_EXTENSIONS = ['webm', 'mp4', 'ogv', 'ogg', 'mov']
```

Comment:
Just read a bit more into this
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

it sounds like not all browsers support the same things and you can use multiple <source> elements to fallback between formats. Since the API results are so hairy, i think what we have here is good